app/controllers/friendships_controller.rb

```diff
@@ -0,0 +1,24 @@
+class FriendshipsController < ApplicationController
+  def create
+    @target_user = User.find_by_email(params[:friendship][:email])
+
+    unless @target_user.nil?
+      @friendship = Friendship.new(user_id: current_user.id, friend_id: @target_user.id)
+      @friendship.save
+    end
+
+    redirect_to :my_friends
+  end
+
+  def my_friends
+    @friendships = current_user.friendships.all
+    @requests = Friendship.where('friend_id = ? AND accepted = 0', current_user.id)
```

Comment:
Line is too long. [83/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.